tests/functional/simple_seed/test_simple_seed.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import pytest
+from dbt.tests.adapter.simple_seed.test_seed import SeedConfigBase
+
+from pathlib import Path
+from dbt.tests.util import (
+    mkdir,
+    rm_dir,
+    run_dbt,
+    read_file
+)
+
+class TestSimpleBigSeedBatched(SeedConfigBase):
```

Comment:
This could have been inherited from [TestSeedSpecificFormats](https://github.com/dbt-labs/dbt-core/blob/c38540af3c830f5e3244c559dc3ebf8278829f3b/tests/adapter/dbt/tests/adapter/simple_seed/test_seed.py#L258) but it fails for `"seed.with.dots.csv": seed__with_dots_csv`